todo_app/templates/view_model.py

```diff
@@ -0,0 +1,34 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def todo_items(self):
+        todo_items=[]
+        for item in self._items:
+            if item.status == 'todo':
```

Comment:
This only works with your unit test, not with the real Trello cards.

And if your index.html actually used `view_model.todo_items` instead of `view_model.items` you would see your integration test fail.

The real statuses you should check for are "To Do", "Doing", "Done". Spaces and capitalisations affect equality checks.